karma.conf.js

```diff
@@ -1,16 +1,40 @@
 // Karma configuration
 
+const { readFileSync } = require('fs');
 const webpackConfig = require('./webpack.config.test');
 
 // Set BABEL_ENV to use proper preset config
 process.env.NODE_ENV = 'test';
 process.env.BABEL_ENV = 'test';
 
 module.exports = function (config) {
+
+  let specsToRetry;
+  if (config.retryFailed) {
+    // Loads names of failed specs and prepares them for use in a regex.
```

Comment:
`karma start --retry-failed` -> `config.retryFailed` is truthy
`karma start` -> `config.retryFailed` is undefined